### DISCUSSION TITLE
Test in iOS device

### DISCUSSION THREAD

**Discussion by mapsouza** [microsoft/dotnet-podcasts#26] [Q&A]
Hi...

I tried to deploy the app to my iOS device without success. When the app tries to connect to API server, it receives (Local network prohibited).

For the test, I tried to put my local IP address in the Config.cs class and add NSAllowsLocalNetworking to Info.plist file. 

Anyone can help me?

**Comment by jamesmontemagno** [comment]
For an iOS deployment you would have to use the simulator to test localhost. If you are on a device you will need to deploy the backend to azure and enter the IP address in there.
